test/cpp/interop/xds_interop_client.cc

```diff
@@ -513,11 +515,13 @@ void RunServer(const int port, StatsWatchers* stats_watchers,
   LoadBalancerStatsServiceImpl stats_service(stats_watchers);
   XdsUpdateClientConfigureServiceImpl client_config_service(rpc_configs_queue);
 
+  grpc::reflection::InitProtoReflectionServerBuilderPlugin();
   ServerBuilder builder;
   builder.RegisterService(&stats_service);
   builder.RegisterService(&client_config_service);
   builder.AddListeningPort(server_address.str(),
                            grpc::InsecureServerCredentials());
+  grpc::AddAdminServices(&builder);
```

Comment:
nit: Would it be better to move it up to after the `builder.RegisterService`? After all, this is just another a service registration.

Reply:
Right. Done.